test/Microsoft.Azure.Webjobs.Extensions.Storage.UnitTests/Blobs/Bindings/WatchableCloudBlobStreamTests.cs

```diff
@@ -2160,7 +2160,7 @@ public async Task CheckLogs_AfterWrite_ReturnsBlobWriteAccessLog()
             Assert.Equal("BlobWriteAccess", logMessage.EventId.Name);
             Assert.Equal(LogLevel.Debug, logMessage.Level);
             Assert.Equal(6, logMessage.State.Count());
-            Assert.Equal($"{containerName}/{blobName}", logMessage.GetStateValue<string>("name"));
+            Assert.Equal($"{containerName}/{blobName}", logMessage.GetStateValue<string>("blobName"));
```

Comment:
Should we add a check for name or function name explicitly?

Reply:
This logger scope does not have functionName.  I think any logs written with in the context of a function execution will have functionName populated. Tagging @brettsam to confirm

Reply:
Right, the scope gets added only during function execution. There should be an existing e2e test that uses blobs that you could add another verification step to. Maybe `AsyncChainEndToEnd`? It checks logs already and looks like it uses blob bindings.